src/main/java/Duke.java

```diff
@@ -1,10 +1,179 @@
+import java.util.Scanner;
+
 public class Duke {
+    private static Task[] tasks = new Task[100]; // Store up to 100 tasks.
```

Comment:
Readers might not be sure what 100 means. You can extract it as a constant (static final) and name it as MAX_TASK_COUNT

Reply:
Will make this suggestion, thanks for it!